rskj-core/src/test/java/co/rsk/logfilter/BlocksBloomTest.java

```diff
@@ -11,7 +11,7 @@ class BlocksBloomTest {
 
```

Comment:
Maybe it would be nice to create a test with the backward logic, and also to very it works the same just to prevent future problems.

Reply:
Actually I found a bug thanks to this.